public/js/wyr-single.js

```diff
@@ -1,14 +1,12 @@
 // Existing code
 const commentBtn = document.querySelector("[name='comment-btn']");
-const closeModal = document.getElementById("close-modal");
+const addCommentCloseModal = document.getElementById("addComment-close-modal");
```

Comment:
I apologize, but I had to change this to be specific with which closing button closed which modal